undebt/cmd/main.py

```diff
@@ -102,23 +83,6 @@ def _find_files(paths, extensions):
         if os.path.isfile(path):
             yield path
 
-        else:
```

Comment:
I am ok with letting it error out.
However, if we want to check it, how about do something like:

if not all([os.path.isfile(f) in args.files]):
    # raise some exception